### PR TITLE
Open in azure portal

### PR DESCRIPTION
**Pull request by aaomidi**

### CODE REVIEW

File: extensions/azurecore/package.json [hunk 1]
```diff
@@ -259,6 +263,11 @@
           "command": "azure.resource.startterminal",
           "when": "viewItem == azure.resource.itemType.account",
           "group": "azurecore"
+        },
+        {
+          "command": "azure.resource.openInAzurePortal",
+          "when": "isDatabaseOrServer",
```
Comment (aaomidi) [inline extensions/azurecore/package.json:269]: @Charles-Gagnon @abist I'm still really bad when it comes to this. What's a good setting here and for the group?

We want this to only show up if azureTenantId and azureResourceId is defined on a connection profile

Comment (abist) [reply extensions/azurecore/package.json:269]: There are 2 cases here. The azureTenantId and azureResourceId is set - 
1. compile time - then you can just use a boolean which is defined in the class
2. dynamically in run time - use `vscode.commands.executeCommand('setContext', 'azureTenantId', true);`

File: extensions/azurecore/src/azureResource/commands.ts [hunk 2]
```diff
@@ -237,4 +237,23 @@ export function registerAzureResourceCommands(appContext: AppContext, tree: Azur
...
+		if (!treeItem.payload) {
+			return;
+		}
+		let connectionProfile = Object.assign({}, treeItem.payload, { saveProfile: true });
+
+		if (!connectionProfile.azureResourceId) {
+			return;
+		}
+
+		const urlToOpen = `https://portal.azure.com//${connectionProfile.azureTenantId}/#resource/${connectionProfile.azureResourceId}`;
```
Comment (Charles-Gagnon) [inline extensions/azurecore/src/azureResource/commands.ts:256]: extra / here after portal.azure.com?

File: extensions/azurecore/package.json (hunk 1 above)
Comment (Charles-Gagnon) [reply extensions/azurecore/package.json:269]: Not sure what you mean by "what a good setting here" is. But from what I think you're saying you would likely be looking to modify https://github.com/microsoft/azuredatastudio/blob/master/src/sql/workbench/services/objectExplorer/browser/mssqlNodeContext.ts#L31

Comment (aaomidi) [reply extensions/azurecore/package.json:269]: Yeah thinking on this, there isn't really a way for this option to show up without this information being populated anyway.

File: extensions/azurecore/src/azureResource/commands.ts [hunk 3]
```diff
@@ -237,4 +237,23 @@ export function registerAzureResourceCommands(appContext: AppContext, tree: Azur
...
+		if (!treeItem.payload) {
+			return;
+		}
+		let connectionProfile = Object.assign({}, treeItem.payload, { saveProfile: true });
+
+		if (!connectionProfile.azureResourceId) {
+			return;
+		}
+
+		const urlToOpen = `${connectionProfile.azurePortalEndpoint}//${connectionProfile.azureTenantId}/#resource/${connectionProfile.azureResourceId}`;
```
Comment (aaomidi) [inline extensions/azurecore/src/azureResource/commands.ts:256]: The // is expected here

**Comment by aaomidi** [comment]
image

**Comment by aaomidi** [comment]
Test failures are unrelated to PR

**Pull request merged**